README.md

```diff
@@ -1,5 +1,10 @@
 # Charmed Kafka K8s Operator
 
+[![CharmHub Badge](https://charmhub.io/kafka-k8s/badge.svg)](https://charmhub.io/kafka-k8s)
+[![Release](https://github.com/canonical/kafka-k8s-operator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/kafka-k8s-operator/actions/workflows/release.yaml)
+[![Tests](https://github.com/canonical/kafka-k8s-operator/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/canonical/kafka-k8s-operator/actions/workflows/ci.yaml?query=branch%3Amain)
```

Comment:
I don't think `?branch=main` is needed. If nothing is specified it will automatically target `default` branch, main in this case.

Reply:
I actually took it from initial @delgod 's PR [here](https://github.com/canonical/mysql-operator/pull/356/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4). 

@delgod what's your take?

Reply:
I want to see the results of night tests on the central dashboard.
after a discussion with Carl, they suggested adding a filter to avoid noise from PR tests.